package/yast2-packager.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Tue Aug 23 11:17:36 UTC 2016 - jreidinger@suse.com
+
+- always enable next in license confirmation dialog, because even
+  if not accepted, popup is shown to user (bnc#993530)
+- use same popup when license not confirmed for base product in
```

Comment:
NP: definite articles: *the same popup* and *in the license confirmation dialog*